example-app/src/admin/resources/photo/photo.resource.ts

```diff
@@ -14,7 +14,7 @@ const createPhotoResource = (): CreateResourceResult<typeof Photo> => ({
     },
   },
   features: [uploadFeature({
-    credentials,
+    provider: { aws: credentials },
```

Comment:
changed that to provider which will also support new adapters in the future. There is also a BaseProvider class with 3 simple methods which should be implemented for all other providers.